ddtrace/contrib/unittest/patch.py

```diff
@@ -34,30 +50,93 @@ def get_version():
 
 
 def _set_tracer(tracer):
+    """Manually sets the tracer instance to `unittest.`"""
     unittest._datadog_tracer = tracer
 
 
-def _store_span(item, span):
+def _store_test_span(item, span):
     """Store span at `unittest` instance."""
     item._datadog_span = span
 
 
+def _store_module_identifier(test_object):
+    """Store module identifier at `unittest` module instance."""
+    if hasattr(test_object, "test") and hasattr(test_object.test, "_tests"):
+        for module in test_object.test._tests:
+            if len(module._tests) and _extract_module_name_from_module(module):
+                _set_identifier(module, "module")
+
+
+def _store_suite_identifier(module):
+    """Store suite identifier at `unittest` suite instance."""
+    if hasattr(module, "_tests"):
+        for suite in module._tests:
+            if len(suite._tests) and _extract_module_name_from_module(suite):
+                _set_identifier(suite, "suite")
+
+
+def _is_test_module(item):
+    return hasattr(item, "_datadog_object") and item._datadog_object == "module"
+
+
+def _is_test_suite(item):
+    return hasattr(item, "_datadog_object") and item._datadog_object == "suite"
+
+
+def _is_test(item):
+    if type(item) == unittest.TestSuite or not hasattr(item, "_testMethodName"):
+        return False
+    if ddtrace.config.unittest.strict_naming and not item._testMethodName.startswith("test"):
+        return False
+    return True
+
+
 def _extract_span(item):
     """Extract span from `unittest` instance."""
     return getattr(item, "_datadog_span", None)
 
 
-def _extract_test_method_name(item):
+def _extract_command_name_from_session(session):
+    """Extract command name from `unittest` instance."""
```

Comment:
This docstring doesn't add any information that isn't contained in the function name.